core/include/drivers/imx/dcp.h

```diff
@@ -0,0 +1,213 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright 2020 NXP
+ */
+#ifndef __IMX_DCP_H__
+#define __IMX_DCP_H__
+
+#include <compiler.h>
+#include <tee_api_types.h>
+#include <types_ext.h>
+#include <util.h>
+
+#define DCP_SHA_BLOCK_SIZE     64
+#define DCP_AES_BLOCK_SIZE     64
+#define DCP_AES128_KEY_SIZE    16
+#define DCP_AES128_IV_SIZE     16
+#define DCP_AES128_BUFFER_SIZE 16
+
+enum dcp_key_mode {
+	SRAM0 = 0,
```

Comment:
Should have at least a `DCP_` prefix to these enum values, same for the others below.

Reply:
I will prepend ```DCP``` to the following enum values.